adapters/sspBC/sspbc_test.go

```diff
@@ -0,0 +1,20 @@
+package sspBC
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderSspBC, config.Adapter{
+		Endpoint: "http://ssp.wp.test/bidder/"})
```

Comment:
There is no need to use a real endpoint here. You may use a fake endpoint to avoid needing to keep the tests in sync with the real url if it changes in the future.

Reply:
Thanks. I think it is already a fake URL. Can it remain in this form?